quickstarts/mlops/mona/config.yml

```diff
@@ -0,0 +1,60 @@
+# TODO(elie): Ask for our id.
+id: d0e829a6-ede4-4933-9065-9f0c56f8aa7e
+
+# Name of the quickstart (required)
+name: mlops-monalabs
+
+# Displayed in the UI
+title: Mona Labs
+
+# Long-form description of the quickstart (required)
+description: |
+  Mona provides the best visibility into AI systems in order to reduce associated risks with production AI, optimize operational processes, and enable teams to plan better AI roadmaps. The Mona - New Relic integration provides users with the ability to automatically view Mona generated insights directly on their New Relic dashboard, explore production AI / ML data with NRQL, and connect specific Mona insight events to New Relic Incident Intelligence.
+
+  Instantly get alerted on latency issues and automatically detect anomalies to ensure the best performance within production systems.
+
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Mona’s integration with New Relic provides next-level monitoring capabilities under a unified production AI environment.
+
+
+# TODO(elie): Make sure this correct.
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - Mona Labs
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml
+  - ai
+  - mona
+  - machine-learning
+  - machine learning
+  - monitoring
+  - data monitoring
+  - data
+  - integrity
+  - data integrity
+  - drift
+  - explainability
+
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - monalabs-mlops
+
+documentation:
+  - name: Mona Labs Docs
+    # TODO(elie): Update with the correct link.
+    url: https://docs.newrelic.com/docs/integrations/mlops-integrations/algorithmia-mlops-integration/
```

Comment:
Is mona and alogorthmia the same thing?

Reply:
please use the proper YAML formatting >- for documentation URL references.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation